packages/core/lib/commands/create/templates/example.js

```diff
@@ -1,9 +1,8 @@
 const Example = artifacts.require("Example");
 
-contract("Example", function() {
-  it("should assert true", async function(done) {
+contract("Example", () => {
```

Comment:
Technically, mocha says you shouldn't use arrow functions here - cause they make use of `this` internally (`this.setTimeout()`, if you wanted to use it)

But this doesn't matter here.

Reply:
Oh, I didn't consider this. I'll change it back to conform to Mocha's docs.

Reply:
I take it back - it does matter here. We don't want to mislead people, since this is a template.

Reply:
adding [the mocha issue for reference](mochajs/mocha/issues/2018).

Reply:
++ template reference. using `function(){...}` callback.